CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 4.0.2
```

Comment:
Sorry, I should have told you: I usually just use `## Master` for entries not released yet
_(I might not release a new version (4.0.2) _just_ for this, and in case the next PR adds something breaking the next version might not end up being 4.0.2 but maybe 4.1.0 or whatnot 😉 )_